MLP pbmc run on HPC (#49)
WayScience/pyroptosis_signature_data_analysis: 4.sc_Morphology_Neural_Network_MLP_Model/MLP_utils/README.md

# MLP_utils
## `Exceptions.py`
This file contains the exceptions rules for the MLP model.

## `parameters.py`

> This file contains the parameters dataclass used for the mlp_params in:

Reviewer: ```suggestion
This module contains the parameters dataclass used for the mlp_params in:
```

* Hyperparameter Optimization
* Training
* Testing

## `utils.py`

> This file contains the utility/helper functions for the MLP model.

Reviewer: ```suggestion
This `utils` module contains helper functions for the MLP model.
```

## trained_models

> This directory contains the trained models from the hyperparameter optimization and training.

Reviewer: ```suggestion
This directory contains trained models generated from the hyperparameter optimization and training.
```
